.chloggen/instana-exporter-https-option.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
I think this is more a bug than an enhancement as it should have been HTTPS only. What do you think?